### DISCUSSION TITLE
Mail sending going in soft fail after enabling the ip pool

### DISCUSSION THREAD

**Discussion by osshaheen** [postalserver/postal#2344] [Help with using Postal]
when I use the IP pool my messages get in soft fail.
The ip is appear when i try command ip a

**Comment by osshaheen** [comment]
@willpower232 Can you suggest this please?

**Comment by willpower232** [comment]
This is the error message you normally see when your server provider is blocking outgoing emails. You should confirm with them whether this is the case.

**Reply by osshaheen** [reply]
Thank you but it was sending before activate the ip pool feature if i remove the new ip from the selected ip pool it will send successfully
